src/routes/reservations.php

```diff
@@ -0,0 +1,164 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: chenyi
+ * Date: 2018-11-12
+ * Time: 4:58 PM
+ */
+
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+
+$app->get('/api/reservations/{username}', function (Request $request, Response $response, array $args) {
```

Comment:
should use `/api/reservations/customer/{username}`, the original URL is very confusing eg: `/api/reservations/1` this is more like the reservation id is 1, not the username=1.

Reply:
also please add a brief comments of what the api do. Easy for front-end guy to know what endpoint to use without reading the code. Could as simple as: `Get a customer's all reservation`